### DISCUSSION TITLE
Showing chord diagram above staff

### DISCUSSION THREAD

**Discussion by ldupouey** [CoderLine/alphaTab#1706] [Q&A] [answered]
Hi,
Is there a way to display chord diagrams directly above the staff and not only at the beginning of the score ?
I'd like to achieve something like this :
![image](https://github.com/user-attachments/assets/08dfbfca-38f1-45e1-9d4f-250c44008e68)
Regard, 
Leo

**Comment by Danielku15** [comment, marked as answer]
Its currently not supported.
We have it on the roadmap to also have inline chord diagrams https://github.com/orgs/CoderLine/projects/12/views/1?filterQuery=dia&pane=issue&itemId=21407298
